schema/i18n.json

```diff
@@ -0,0 +1,112 @@
+"interview_question": {
+  "title": "Internationalization (i18n)",
+  "prompts": [
+    {
+      "instructions": "Welcome to this programming exercise.\n\nYou will complete this exercise by editing the code in the sample.py file\nusing your favorite editor.\nEvery time you save the file, I will evaluate your code and let you know about\nnext steps.\n\nYour first task is to implement the abbreviate function. It takes a string as input\nand return an abbreviation of the string of the form <first character><length of the\nmiddle of the string><last character>. For example, \"internationalization\" should be\nabbreviated as \"i18n\".",
```

Comment:
Do you need the intermediate newlines, e.g. between 'file' and 'using'? I thought we could just trust the UI to handle wrapping, and you'd only need paragraph breaks.

Speaking of paragraph breaks, if you want those too, would it be better to have a list of strings?